webpack.config.js

```diff
@@ -10,19 +11,38 @@ const config = {
     path: BUILD_DIR,
     filename: 'bundle.js'
   },
-  resolve: {
-    extensions: ['.js', '.jsx']
-  },
   module: {
     loaders: [
       {
-        test: /\.jsx?$/,
+        test: /(\.js|\.jsx)$/,
 
-        include: APP_DIR,
+        include: [
+          APP_DIR,
+          path.join(__dirname, 'node_modules/react-codemirror')
+        ],
         loader: 'babel-loader'
+      },
+      {
+        test: /(\.scss|\.css)$/,
+        loader: ExtractTextPlugin.extract({
+          fallback: 'style-loader',
+          use: 'css-loader'
```

Comment:
Perhaps we can make a note in backlog to look into this later on and make this loader work instead of using our own styles.css. Understand it's not a priority.